src/map/ai/ai_container.cpp

```diff
@@ -169,6 +170,19 @@ void CAIContainer::Trigger(uint16 targID)
     }
 }
 
+void CAIContainer::UseItem(uint16 targid, uint8 loc, uint8 slotid)
+{
+    auto PlayerController = dynamic_cast<CPlayerController*>(PEntity->PAI->GetController());
+    if (PlayerController)
+    {
+        PlayerController->UseItem(targid, loc, slotid);
+    }
+    else
```

Comment:
remove this else, I changed it recently where I don't want it to run without a controller (jailed char = remove controller)